posts/2017-04-12-serverless-v1.11.0.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Separate Package and Deploy command, CloudWatch Log event support with Serverless v1.11
```

Comment:
Just thought about this problem again. The issue here is that both topics are really interesting and release highlights.

The `package` and `deploy` separation was a super high priority change requested by a bunch of different enterprise users to make CI / CD better.

The `cloudwatchLog` event source on the other hand is super interesting for day-to-day Serverless users 🤔